src/constants/main.ts

```diff
@@ -7,12 +7,11 @@ export const APP_NAME = "Bot Busters";
 
 export const CHAT_TIME_MS = 180000; // 3 minutes
 export const MATCH_TIME_MS = 600000; // 10 minutes
-export const ALERT_TIME_MS = 30000;
-export const VOTING_TIME_MS = 60000;
-
-export const SPLASH_SCREEN_DURATION = 6500; // 6.5 seconds
-export const SPLASH_SCREEN_TEXT_DURATION = 500; // 0.5 seconds
-export const MATCHMAKING_DELAY = 1800;
+export const ALERT_TIME_MS = 30000; // 30 seconds
+export const VOTING_TIME_MS = 60000; // 1 minute
+export const SPLASH_SCREEN_TIME_MS = 6500; // 6.5 seconds
+export const SPLASH_SCREEN_TEXT_DELAY = 500; // 0.5 seconds
```

Comment:
Do you want to add `_MS` to every const for consistency?
Or remove it from all and we all understand that we're always talking about milliseconds.